idris2.ipkg

```diff
@@ -1,182 +1,6 @@
-package idris2
+package idris2app
```

Comment:
I would probably keep the package name defined by `idris2.ipkg` to be `idris2`
and make `idris2api.ipkg` define `idris2api` instead.

Reply:
This package name will never be used as a dependency or really anywhere, the lib will, so it gets the `idris2` name.

Reply:
Maybe, then, rename `idris2api.ipkg` to `idris2lib.ipkg` and `idris.ipkg` to `idris2app.ipkg`?

Reply:
That's probably better, but I want to keep these changes unobtrusive.